### DISCUSSION TITLE
Production ready?

### DISCUSSION THREAD

**Discussion by V01D-NULL** [tmc/langchaingo#74] [Q&A] [answered]
Hey!

Great project.
langchain and go are a fantastic combination.

I like the idea so much that I'm considering proposing this to my team since we're looking into langchain for production usage. 

Go feels like it's the right way to go, but I am unfamiliar with how far behind(or ahead :D) this project is compared to lanchain or langchainjs.

I'd appreciate some feedback on this.

Thanks!

Edit:

I saw https://github.com/tmc/langchaingo/issues/61, but I just wanted to double-check that this is actually the state of langchaingo or if that was just an example.

**Comment by Struki84** [comment, marked as answer]
Here's a parity matrix, tracks how far behind go version is, but I'm not sure is it up to date... https://github.com/tmc/langchaingo/blob/main/docs/parity_matrix.md

**Comment by tmc** [comment]
We're certainly younger and lagging (for now) in terms of the feature matrix but we welcome contributions. I have deployed several internet-facing systems that use parts of this codebase.
